pom.xml

```diff
@@ -75,7 +75,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <!-- Dependencies -->
-        <okhttp.version>4.2.2</okhttp.version>
+        <okhttp.version>3.12.6</okhttp.version>
```

Comment:
Shall we upgrade to `3.14.7` instead?

Reply:
It **must** be the same as the one in Kubernetes-Client. If I'm not mistaken (please double check), this is 3.12.6.

Reply:
OkHttp semver is not very good, so it relies in internal classes that have been moving from one package to another. So mixing versions causes failures such as the ones mentioned.

Reply:
Sorry, I meant upgrading versions at both place to `3.14.7`.

Reply:
We can, but I'm currently blocked on this one.
Hopefully after upgrading the version here, and in JKube, and releasing (only mockwebserver) I can continue with my work.

We can do the global version upgrade then, but for me it would be very helpful to fix this one ASAP.